.github/workflows/ci.yml

```diff
@@ -170,180 +81,41 @@ jobs:
           curl -L "$LINK/$SCCACHE_VERSION/$SCCACHE_FILE.tar.gz" | tar xz
           mv -f $SCCACHE_FILE/sccache $HOME/.local/bin/sccache
           echo "$HOME/.local/bin" >> $GITHUB_PATH
-      - name: Install sccache - macos-latest
+      - name: Install sccache (macos-latest)
         if: matrix.os == 'macos-latest'
         run: brew install sccache
-      - name: Install ${{ matrix.rust }}
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ matrix.rust }}
-          override: true
-      - name: Cache cargo registry
-        uses: actions/cache@v2
-        continue-on-error: true
-        with:
-          path: ~/.cargo/registry/cache
-          key: ${{ matrix.cargo-registry-cache-key-part }}-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: ${{ matrix.cargo-registry-cache-key-part }}-
-      - name: Cache sccache output
-        uses: actions/cache@v2
-        continue-on-error: true
-        with:
-          path: ${{ matrix.sccache-path }}
-          key: ${{ matrix.sccache-key-part }}-${{ hashFiles('**/Cargo.*') }}
-          restore-keys: ${{ matrix.sccache-key-part }}-
-      - name: Start sccache server
-        run: sccache --start-server
-      - name: Run doc tests
-        run: make run-all-doc-test
-      - name: Stop sccache server
-        run: sccache --stop-server
-
-  unit_test_mac:
-    name: Unit Test Mac
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [macos-latest]
-        rust: [stable]
-    env:
-      RUST_BACKTRACE: full
-      RUSTC_WRAPPER: sccache
-      SCCACHE_CACHE_SIZE: 300M
-      SCCACHE_DIR: /Users/runner/Library/Caches/Mozilla.sccache
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Install sccache
-        run: brew install sccache
-      - name: Install ${{ matrix.rust }}
+      - name: Install Rust ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
+          profile: minimal
           override: true
       - name: Cache cargo registry
         uses: actions/cache@v2
-        continue-on-error: true
-        with:
-          path: ~/.cargo/registry/cache
-          key: test-${{ runner.os }}-${{ matrix.conf }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            test-${{ runner.os }}-${{ matrix.conf }}-cargo-registry-
-      - name: Cache sccache output
-        uses: actions/cache@v2
-        continue-on-error: true
```

Comment:
Can you remove verbose?
```
 Running `sccache rustc --crate-name cfg_if --edition=2018 /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/cfg-if-1.0.0/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=1 -C metadata=7be9226c66ab2048 -C extra-filename=-7be9226c66ab2048 --out-dir /Users/runner/work/fluvio/fluvio/target/debug/deps -L dependency=/Users/runner/work/fluvio/fluvio/target/debug/deps --cap-lints allow`
```

Reply:
Problem is that it is hard to look at unit test log because SCCACHE log was filling up output.  How about option to set verbose flag when invoked by `workflow_dispatch`

Reply:
I updated the Makefile to append the value of $(VERBOSE) to build tasks and I set the workflow dispatch up with an input for a "verbose" string that you can fill with `--verbose` in order to make it run in  verbose mode. So now by default things will run without verbose but you can use the workflow dispatch input to set it